interactive_templates/templates/v2/analysis/render_report.py

```diff
@@ -88,7 +88,10 @@ def get_data(
         dict containing the data
     """
 
-    breakdowns = breakdowns.split(",")
+    if breakdowns != "":
+        breakdowns = breakdowns.split(",")
+    else:
+        breakdowns = []
```

Comment:
If you instead wanted to change breakdowns to an nargs action="append", which by default would be an empty list, you could change the jinja template to render something like this:

>   generate_measures_{{ id }}:
    run: >
      python:latest -m analysis.measures
      {% for demo in demographics -%}
        --breakdowns={{demo}}
      {% endfor -%}
        --input_dir="output/{{ id }}/joined"

Reply:
It would be good to do this. We have to pass the breakdowns in as a string for the cohortextractor action so this just copies that, but agree it's not very nice. I couldn't get my head around templating the args in `project.yaml.tmpl` with nargs, do you have any ideas?